regex-assembly/932281.ra

```diff
@@ -0,0 +1,20 @@
+##! Please refer to the documentation at
+##! https://coreruleset.org/docs/development/regex_assembly/.
+
+##! Spaces are not allowed in brace expansions.
```

Comment:
This file should have the same comment as the other one. In addition, the comment should describe the difference to the expression generated from the other file.